test/wrapper_test.py

```diff
@@ -1,15 +1,17 @@
+#!/usr/bin/python
 # -*- coding: utf-8 -*-
+
 import inspect
+import logging
```

Comment:
I'm guessing you ran isort on this and cleaned up imports? If possible it is best to do purely cosmetic changes like that in seperate PRs, fine as is for now, but if something is purely cosmetic it gets through a lot quicker, and I think we can adjust our internal guidelines to say that for cosmetic only changes one reviewer is enough.

Reply:
I'm going to try setup https://pre-commit.ci/ also for us on RDFLib/rdflib sometime - that will also make it easier to periodically clean these things up automatically.

Reply:
OK, will this do things like run `black`? And perhaps other code beautifying things?

Reply:
yes, and we can make it run isort, and pycln to clean up uneeded imports

Reply:
will try do it this week still